package.json

```diff
@@ -9,6 +9,7 @@
     "lint:scss": "stylelint scss/**/*.scss --syntax scss",
     "lint:js": "eslint --ignore-path .gitignore --ext js,jsx .",
     "lint": "yarn lint:scss && yarn lint:js",
+    "test": "sass test/scss:test/scss/tmp --style=compressed --no-source-map && sass scss/bitstyles.scss:test/scss/tmp/bitstyles.css --style=compressed --no-source-map && postcss test/scss/tmp/*.css --replace && npx prettier --write test/scss/tmp/*.css && diff --from-file test/scss/fixtures/bitstyles-overrides.css test/scss/tmp/test-import-all.css && diff --from-file test/scss/fixtures/bitstyles-overrides.css test/scss/tmp/test-use-all.css && diff --from-file test/scss/fixtures/bitstyles-overrides.css test/scss/tmp/test-import-each.css && diff --from-file test/scss/fixtures/bitstyles.css test/scss/tmp/bitstyles.css",
```

Comment:
Yes, let's turn this into a bash script. Easier to format and therefore easier to read? Do you need help/inspiration for how to do it? I can also take over if you feel it's taking you too long.

Inspiration can be found at e.g. https://github.com/bitcrowd/tickety-tick

Reply:
Thanks! I’ll take a look over there 🙏

Reply:
I’ve updated, this is now handled in a bash script (called via `yarn test` for convenience)